packages/slice-machine/src/modules/slices/index.ts

```diff
@@ -92,6 +92,17 @@ export const getRemoteSlices = (
   store: SliceMachineStoreType
 ): ReadonlyArray<SliceSM> => store.slices.remoteSlices;
 
+export const getUnsyncedSlices = (
+  store: SliceMachineStoreType
+): ReadonlyArray<ComponentUI> => {
+  return store.slices.libraries.reduce((acc, lib) => {
+    const unsycnedComponents = lib.components.filter((component) =>
+      [LibStatus.Modified, LibStatus.NewSlice].includes(component.__status)
+    );
+    return [...acc, ...unsycnedComponents];
+  }, [] as ReadonlyArray<ComponentUI>);
```

Comment:
alternatively, you could pass a type parameter to `reduce` and it should save type casting the inital array.

Reply:
true you already told me about this another time, will change it